test/functionalTest/cases/1308_subscription_cache/subCacheMixedSubscriptionsWithUpdatesAndDeletes.test

```diff
@@ -422,12 +424,12 @@ Date: REGEX(.*)
     "jsonRequests": "5",
     "measuring_interval_in_secs": "REGEX(\d+)",
     "statisticsRequests": "5",
-    "subCache": "REGEX([0-9a-f]{24})",
-    "subCacheInserts": "REGEX((1|2))",
-    "subCacheItems": "REGEX((1|2))",
+    "subCache": "REGEX([0-9a-f, ]{76})",
+    "subCacheInserts": "REGEX((5|8))",
+    "subCacheItems": "3",
```

Comment:
You have colapsed an OR (in the REGEX) to a particular value (maybe the same in other places). This maybe make valgrind suite to cry, but I understand that it is difficult to adjust at this point...